libs/langchain-community/package.json

```diff
@@ -213,6 +213,7 @@
     "youtubei.js": "^9.1.0"
   },
   "peerDependencies": {
+    "@arcjet/redact": "^v1.0.0-alpha.23",
```

Comment:
Should this be more relaxed?

Reply:
Also needs to be under `devDependencies` too but I can add that

Reply:
No that's actually the first version of the package. While we're in alpha we're just using common versioning for all of our npm packages, so `@arcjet/redact` starts with `v1.0.0-alpha.32`

Reply:
It starts with `v1.0.0-alpha.23` not v1.0.0-alpha.32`.